src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1912,6 +1912,32 @@ public function getLongHostNames()
             array(str_repeat(':', 101)),
         );
     }
+
+    /**
+     * @dataProvider methodProvider
+     */
+    public function testMethodSafe($method, $safe)
+    {
+        $request = new Request();
+        $request->setMethod($method);
+        $this->assertEquals($safe, $request->isMethodSafe());
+    }
+
+    public function methodProvider()
+    {
+        return array(
+            array(Request::METHOD_HEAD, true),
```

Comment:
to be consistent with comments on other PRs, shouldn't we use plain strings here?